lastminute.yml

```diff
@@ -276,15 +278,15 @@
         executable: /bin/bash
       become: false
       register: git_revision
-      delegate_to: 127.0.0.1
+      delegate_to: localhost
       changed_when: false
 
     # TODO to be done in icpc2024
-    - name: Stop and disable apport service
-      ansible.builtin.service:
-          name: apport
-          state: "stopped"
-          enabled: false
+    # - name: Stop and disable apport service
+    #   ansible.builtin.service:
```

Comment:
just thinking about more, maybe it would be better to ensure the apport package was not installed?